src/algorithms/helpers/paths-diversity.ts

```diff
@@ -0,0 +1,36 @@
+import Path from '../pathfinders/path';
+
+function getDiversity(shortest: Path, current: Path): number {

```

Comment:
I'd put that as a method into the relevant class... deja vu :D well if it works it works and leave it